policies/new_chapter_policy.md

```diff
@@ -144,6 +144,26 @@ Approved chapters will be created and activated by 15th of each month. The proje
 team lead will create the chapter's email address by this date. Chapter organizers will receive email
 notification as soon as email address has been created.
 
+### Review mechanics
+
+1. When prospective organizers filled in the new chapter request form, their response is stored as a row in google spreadsheet
+
+2. PyLadies Slack Bot will post the form response into #feed-new-chapter Slack channel
+
+3. Each reviewers will read the requests and give vote by adding either :thumbsup: or :thumbsdown: reaction to the Slack post.
+
+4. If there is a :thumbsup: reaction made by one of the review team members, PyLadies Slack bot records the member name, and the timestamp.
+
+5. If there is a :thumbsdown: reaction made by one of the review team members, PyLadies Slack bot will prompt for a reason. Team members will follow up.
+
+6. If :thumbsup: or :thumbsdown: reaction was not made by review team member, PyLadies Slack bot will not do anything.
+
+7. After the 10th of the month and before the 15th of the month, team lead will manually check the spreadsheet to see which ones are approved and create the chapters.
```

Comment:
Can we add a link to who the current team lead is? We have in the README the section about team members so linking to that I htink works!

Reply:
ok I've linked to the team members section.